sdk/machinelearning/azure-resourcemanager-machinelearning/CHANGELOG.md

```diff
@@ -1,14 +1,15 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0 (2023-01-11)
 
-### Features Added
+- Azure Resource Manager Machine Learning client library for Java. This package contains Microsoft Azure SDK for Machine Learning Management SDK. These APIs allow end users to operate on Azure Machine Learning Workspace resources. Package tag package-2022-10. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
 
 ### Breaking Changes
 
-### Bugs Fixed
+#### `models.EndpointPropertiesBase` was modified
 
-### Other Changes
+* `java.net.URL swaggerUri()` -> `java.lang.String swaggerUri()`
+* `java.net.URL scoringUri()` -> `java.lang.String scoringUri()`
```

Comment:
This is due to this [PR](https://github.com/Azure/autorest.java/pull/1881). Now we generate String for uri by default.
@weidongxu-microsoft Guess we can allow this, since it's the first stable version?

Reply:
Yes, this is fine. https://azure.github.io/azure-sdk/java_introduction.html#java-api-url (Srikanta's change)

Reply:
If any GA version got this, we can add `url-as-string: false`

Reply:
Got it. I'll allow it to align with current SDK best practice, and it's their first GA version.